index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portofolio</title>
+    <link rel="stylesheet" href="index.css" />
```

Comment:
- [ ] I see you are using index.css but your css file name is style.css. This will not make your css take effect. Kindly change the css name